addOns/soap/src/main/java/org/zaproxy/zap/extension/soap/ExtensionImportWSDL.java

```diff
@@ -47,7 +44,7 @@
 import org.zaproxy.zap.model.ValueGenerator;
 import org.zaproxy.zap.view.ZapMenuItem;
 
-public class ExtensionImportWSDL extends ExtensionAdaptor {
+public class ExtensionImportWSDL extends ExtensionAdaptor implements SessionChangedListener {
```

Comment:
Nested class (or anonymous) would be better than exposing through the extension.